src/handbook/customer/marketing/tradeshow.md

```diff
@@ -1,22 +1,149 @@
 ---
-navTitle: Marketing
+navTitle: Trade Shows
 navGroup: Sales & Marketing
 ---
 
-# Trade show
+# Trade Shows  
 
-## As exhibitor
+Trade shows are valuable opportunities to connect face-to-face with our industry—customers, partners, and potential users alike. They give us the chance to showcase our product in action, gather feedback, and build relationships that don’t always happen through online calls. Participating in a trade show is both a marketing initiative and a team effort, and often represents a significant investment.
 
-### Booth design
 
-The design of the booth is imporant. From the angles that visitors walk past the
-booth it's vital to have the main logo's in sight: FlowFuse and Node-RED. If the
-space is large enough, ensure an interactive element is added to the booth for
-visitors to use and get an idea of what FlowFuse offers.
+## Preparing for a Trade Show as an Exhibitor  
 
-## As visitor
+Planning a trade show well in advance is crucial. From aligning on the goals and messaging, to making sure we have the right hardware, giveaways, and booth setup—it’s a team effort. We break this down into five areas:
 
-### Preparation before the fair
+1. **Logistics** – budget, team selection, travel, accommodation, vendors, equipment and giveaways.
+2. **Demo** – what we showcase during the event.
+3. **Marketing and Sales** – messaging, promotion and lead generation and nurturing.
+5. **Design** – booth layout and artworks.
+6. **On-site team** – prep for the event and etiquette.
+
+### Logistics
+
+After the decision has been made to attend an event, the first thing that's needed is a budget. This will guide the next steps in the event planning. The CEO is responsible for setting the budget and it should be tracked in a spreadsheet alongside all the logistic details, such as hotel details, team arrival and departure times, plus any relatet matter and added to the relevant folder in the Drive inside [Fairs](https://drive.google.com/drive/folders/1kfbJy6H3DXYlKDd0RZx-psK7ghDmN4BI?usp=drive_link).
+
+**The planning should start at least six months in advance**, so there's enough time to book hotels and for the team to block their agendas. The on-site team should be proposed by the CEO and CTO, and in general, the team will be of 4 or 5 members and composed of:
```

Comment:
Size depends on the event I think. Like the Mayker event we'll likely send one or two.